tests/next_tests/integration_tests/feature_tests/ffront_tests/test_gt4py_builtins.py

```diff
@@ -41,78 +41,52 @@
 
 
 def test_maxover_execution(unstructured_case):
-    """Testing max_over functionality."""
-
-    if unstructured_case.backend in [
-        gtfn_cpu.run_gtfn or fieldview_backend,
-        gtfn_cpu.run_gtfn_imperative,
-        dace_iterator.run_dace_iterator,
-    ]:
-        pytest.skip("not yet supported.")
+    if unstructured_case.backend == dace_iterator.run_dace_iterator:
+        pytest.xfail("Not supported in DaCe backend: reductions")
 
     @gtx.field_operator
-    def maxover_fieldoperator(
-        inp_field: gtx.Field[[Vertex, V2EDim], int32]
-    ) -> gtx.Field[[Vertex], int32]:
-        return max_over(inp_field, axis=V2EDim)
-
-    inp = gtx.np_as_located_field(Vertex, V2EDim)(unstructured_case.offset_provider["V2E"].table)
+    def maxover_fieldoperator(inp_field: cases.EField) -> cases.VField:
```

Comment:
TODO: maxover is failing in C++ because we generate `-2147483648` which in c++ is `long int` because `2147483648` is a long int literal which is then negated.
2 possible solution:
- add a `static_cast<>` around literals
- add numeric limits into our language